packages/fast-components-react-msft/src/flyout/flyout.stories.tsx

```diff
@@ -0,0 +1,183 @@
+import { storiesOf } from "@storybook/react";
+import React, { useState } from "react";
+import { action } from "@storybook/addon-actions";
+import {
+    Flyout,
+    FlyoutHorizontalPosition,
+    FlyoutProps,
+    FlyoutVerticalPosition,
+} from "./";
+import { FlyoutAxisPositioningMode } from "./flyout.props";
+import { Heading, HeadingSize } from "../heading";
+import { Paragraph, ParagraphSize } from "../paragraph";
+import { AccentButton } from "../accent-button";
+
+interface FlyoutTestState {
+    /**
+     * Conditionally render the flyout
+     */
+    visible: boolean;
+}
+
+class FlyoutTest extends React.Component<FlyoutProps, FlyoutTestState> {
+    private anchor: React.RefObject<any>;
+
+    constructor(props: FlyoutProps) {
+        super(props);
+
+        this.anchor = React.createRef();
+        this.state = {
+            visible: false,
+        };
+    }
+
+    public render(): JSX.Element {
+        return (
+            <React.Fragment>
+                <button
+                    ref={this.anchor}
+                    onClick={this.updateFlyoutState}
+                    style={{ margin: "0 auto" }}
+                >
+                    Flyout anchor
+                </button>
+                {this.renderFlyout()}
+            </React.Fragment>
+        );
+    }
+
+    private renderFlyout(): JSX.Element {
+        if (!this.state.visible) {
+            return;
+        }
+
+        const { anchor, children, disabled, ...props }: FlyoutProps = this.props;
+
+        return (
+            <Flyout
+                anchor={this.anchor.current}
+                disabled={!this.state.visible}
```

Comment:
"disabled" isn't an intuitive name to me to convey whether the flyout is visible or not - should we deviate from the viewport positioner API here?

Reply:
I think we should - visible makes more sense to me. I’ll update the contract.